lib/Factory/ResolutionFactory.php

```diff
@@ -171,6 +171,13 @@ public function query($sortOrder = null, $filterBy = [])
         if ($parsedFilter->getInt('enabled', ['default' => -1]) != -1) {
             $body .= ' AND enabled = :enabled ';
             $params['enabled'] = $parsedFilter->getInt('enabled');
+
```

Comment:
The layout background form that runs this query, always has `enabled = 1` and `withCurrent = <currentResolutionId>`
If I were to do what you suggested, it would always return only current resolution in that dropdown and nothing else.

Anyway, if we want to be strict about it, it needs to be a bit different, I will change it again.

Reply:
So do we just need some brackets? `( AND enabled = :enabled OR `resolution`.resolutionId = :withCurrent )`